plugins/bitbucket-cloud-common/openapitools.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "../../node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "5.4.0",
+    "generators": {
+      "backstage": {
+        "generatorName": "typescript-fetch",
+        "glob": "bitbucket-cloud.oas.json",
+        "output": "src",
+        "additionalProperties": {
+          "disallowAdditionalPropertiesIfNotPresent": false,
+          "enumPropertyNaming": "PascalCase",
+          "legacyDiscriminatorBehavior": false,
+          "modelPropertyNaming": "original",
```

Comment:
As of the coding conventions, camelCase is supposed to be used for properties. Bitbucket Cloud uses snake_case at their APIs. It is possible to change this here by using "camelCase" but would also require to deeply map the keys in the request or response bodies when converting a type into a JSON payload.

For the moment, I opted in for keeping the original naming. Let me know if you prefer camelCase despite the mapping for keys.